### ISSUE TITLE
Pond - Automation of ngsQC table creation in Galaxy

### ISSUE THREAD

**Issue by steph-sing** [FDA-ARGOS/data.argosdb#166]
Start with these 4 Orgs for ngsQC:

**_Notes:_**  
- All Orgs in 1 dataset
- File name of dataset should be: ngsQC_PL.tsv
- Current BCO draft: https://www.biocomputeobject.org/builder/https/biocomputeobject.org/ARGOS_000021/DRAFT
- Current Data dictionary: https://github.com/FDA-ARGOS/data.argosdb/tree/v1.1

organism_name: Salmonella enterica LT2
genome_assembly_id: GCA_001558355.2
taxonomy_id: 28901
biosample: SAMN03996249
sra_run_id: SRR2814419

organism_name: Severe acute respiratory syndrome coronavirus 2 Wuhan-Hu-1
genome_assembly_id: GCA_009858895.3
taxonomy_id: 2697049
biosample: SAMN13922059 
sra_run_id: SRR10971381

organism_name: Influenza A virus A/Puerto Rico/8/1934(H1N1)
genome_assembly_id: GCA_000865725.1
taxonomy_id: 211044
biosample: SAMEA51847918
sra_run_id: ERR2096902

organism_name: Marburg marburgvirus Musoke
genome_assembly_id: NA
taxonomy_id: 11269
biosample: SAMN11077998
sra_run_id: SRR10027406

**Comment by stephenshank** [comment]
Steven Weaver's ObservableHQ notebook from September:
https://observablehq.com/@stevenweaver/argos-ngs-qc

I've installed the associated workflow on our instance and refactored it to use collections: https://galaxy.hyphy.org/u/stephenshank/w/ngs-qc-imported-from-uploaded-file-3

I am working on updating to the latest data dictionary, plus utilizing the exciting ObservableHQ/Galaxy integration.

**Comment by steph-sing** [comment]
Moving into Feb 2023 Task list

**Comment by stephenshank** [comment]
I have developed [a tool in Rust to calculate average Phred scores per base](https://github.com/stephenshank/phredmeans).

I've also began porting the above notebook and modified workflow to data dictionary v1.1 and [integrating with Observable](https://observablehq.com/@stephenshank/ngs-qc?payload_id=6cdda2c32937fa91&base_url=galaxy.hyphy.org). This is a WIP and will probably take another day or two of development, but we should have all required fields and a no-code approach.

I noticed that `percent_sequences_classified` was dropped since `v0.8` and would like to request this be restored as an optional column for `v1.2`. We spent some effort getting this field from Kraken and it would be nice to see it utilized.
